flexxamples/ui_usage/combo_box.py

```diff
@@ -0,0 +1,23 @@
+"""
+Simple use of a dropdown,
+"""
```

Comment:
Unfinished docstring? Also some commented code further down. Probably better to either revive or remove.

Reply:
I removed this one. Will add back later if I play more with it. It seem to have issues.